tests/ui/delegation/target-expr.rs

```diff
@@ -3,20 +3,24 @@
 
 trait Trait {
     fn static_method(x: i32) -> i32 { x }
+    fn foo(&self) {}
 }
 
 struct F;
 
 struct S(F);
-impl Trait for S {}
+impl Trait for S {
+    reuse Trait::foo { self.0; }
```

Comment:
Need a comment explaining what is tested here.

Reply:
After changing the body lowering, the test is no longer needed.